src/styles/blocks/card.css

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin-left: 19px;
+  margin-top: 32px;
+}
+
+.card__text-block {
+  margin-top: 40px;
+  margin-left: 16px;
+}
+
+.card__text {
+  font-family: "Roboto", sans-serif;
```

Comment:
move to `html` selector and use `inherit` where needed

Reply:
fix )